api/src/certification/session-management/application/finalize-controller.js

```diff
@@ -24,12 +23,10 @@ const finalize = async function (request, h, dependencies = { certificationRepor
       certificationReports,
     });
 
-    const events = await usecases.processAutoJury({ sessionFinalized });
-    certificationJuryDoneEvents = events.certificationJuryDoneEvents;
-    autoJuryDone = events.autoJuryDone;
+    const { autoJuryDone } = await usecases.processAutoJury({ sessionFinalized });
```

Comment:
Je propose de mettre à plat puisque on ne renvoie plus plusieurs objets 😊 

```suggestion
    const autoJuryDone = await usecases.processAutoJury({ sessionFinalized });
```